src/pages/Login.jsx

```diff
@@ -69,18 +73,19 @@ const LoginScreen = () => {
         });
       })
       .catch((error) =>
-        enqueueSnackbar(error.message, {
+        enqueueSnackbar("Make sure your account is linked to your facebook account", {
```

Comment:
We can't hardcode this error message, there're several error messages left to be handled. For example : Firebase: An account already exists with the same email address but different sign-in credentials. Sign in using a provider associated with this email address. (auth/account-exists-with-different-credential).